src/playlist-loader.js

```diff
@@ -414,6 +416,8 @@ export default class PlaylistLoader extends EventTarget {
         return;
       }
 
+      playlist.lastRefresh = Date.now();
```

Comment:
Should we be unsetting lastRefresh somewhere? Maybe before the request in case if fails?

Reply:
No need, we want to know when the last time we tried to reach this manifest. Maybe `lastRefresh` should be `lastRetry` instead?